src/utilities/theme/utils.ts

```diff
@@ -258,6 +259,29 @@ function successColors(color: HSLAColor, lightSurface: boolean) {
   };
 }
 
+function opacityColors() {
```

Comment:
Pedantic as this is function name is private, but:

```suggestion
function translucentColors() {
```